src/osip-monitoring-controller/src/main/java/edu/kit/pse/osip/monitoring/controller/MonitoringController.java

```diff
@@ -82,12 +124,498 @@ private void setupUI(Stage stage) {
+//        mixCont.client.subscribeFillLevel(interval, value -> mixCont.tank.setLiquid(new Liquid(value * mixCont.tank.getCapacity(), mixCont.tank.getLiquid().getTemperature(), mixCont.tank.getLiquid().getColor())));
+//        mixCont.client.subscribeMotorSpeed(interval, value -> mixCont.tank.getMotor().setRPM(value));
+//        mixCont.client.subscribeOutputFlowRate(interval, value -> mixCont.tank.getOutPipe().setValveThreshold((byte)(value * 100)));
+//        mixCont.client.subscribeOverflowSensor(interval,
+//                value -> mixCont.client.subscribeFillLevel(-1,
+//                        immValue -> mixCont.tank.setLiquid(new Liquid(immValue * mixCont.tank.getCapacity(),
+//                                mixCont.tank.getLiquid().getTemperature(), mixCont.tank.getLiquid().getColor()))));
+//        mixCont.client.subscribeOverheatSensor(interval,
+//                value -> mixCont.client.subscribeTemperature(-1,
+//                        immValue -> mixCont.tank.setLiquid(new Liquid(mixCont.tank.getLiquid().getAmount(), immValue, mixCont.tank.getLiquid().getColor()))));
+//        mixCont.client.subscribeTemperature(interval, value -> mixCont.tank.setLiquid(new Liquid(mixCont.tank.getLiquid().getAmount(), value, mixCont.tank.getLiquid().getColor())));
+//        mixCont.client.subscribeUndercoolSensor(interval,
+//                value -> mixCont.client.subscribeTemperature(-1,
+//                        immValue -> mixCont.tank.setLiquid(new Liquid(mixCont.tank.getLiquid().getAmount(), immValue, mixCont.tank.getLiquid().getColor()))));
+//        mixCont.client.subscribeUnderflowSensor(interval,
+//                value -> mixCont.client.subscribeFillLevel(-1,
+//                        immValue -> mixCont.tank.setLiquid(new Liquid(immValue * mixCont.tank.getCapacity(),
+//                                mixCont.tank.getLiquid().getTemperature(), mixCont.tank.getLiquid().getColor()))));
+//        } catch (UAClientException e){
+//            
+//        }
+        
+        try {
+            mixCont.client.subscribeColor(intervalRegular, mixCont.colorListener);
+            mixCont.client.subscribeFillLevel(intervalRegular, mixCont.fillLevelListener);
+            mixCont.client.subscribeMotorSpeed(intervalRegular, mixCont.motorSpeedListener);
+            mixCont.client.subscribeOutputFlowRate(intervalRegular, mixCont.outputFlowRateListener);
+            mixCont.client.subscribeOverflowSensor(intervalAlarm, mixCont.overflowSensorListener);
+            mixCont.client.subscribeOverheatSensor(intervalAlarm, mixCont.overheatSensorListener);
+            mixCont.client.subscribeTemperature(intervalRegular, mixCont.temperatureListener);
+            mixCont.client.subscribeUndercoolSensor(intervalAlarm, mixCont.undercoolSensorListener);
+            mixCont.client.subscribeUnderflowSensor(intervalAlarm, mixCont.underflowSensorListener);
+        } catch(UAClientException e) {
+            System.err.println(e.getMessage());
```

Comment:
Wie wärs wenn du ausgibst: System.err.println("Could not subscribe property: " + e.getMessage())? Damit kann der Benutzer vermutlich mehr anfangen.